pandas/tests/indexing/test_iloc.py

```diff
@@ -1092,6 +1093,23 @@ def test_iloc_getitem_with_duplicates2(self):
         expected = df.take([0], axis=1)
         tm.assert_frame_equal(result, expected)
 
+    def test_iloc_getitem_interval(self):
```

Comment:
I think you are testing setitem too?

Reply:
Good point, have removed getitem from the name